docs/website/docs/examples/index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Code Examples
+description: A list of comprehensive code examples that teach you how to solve real world problem.
+keywords: ['examples']
+---
+import DocCardList from '@theme/DocCardList';
+
+:::info
```

Comment:
I suggest to swap the order the info admonition and the text below. So text comes first, then info.